src/types.ts

```diff
@@ -1,3 +1,5 @@
+import { optionalSymbol } from './index';
```

Comment:
@lazarljubenovic Is it ok for this file to import?

Reply:
It's not a problem for the file to import, but importing _from_ `index.ts` isn't a good idea because it can lead to issues with circular references. The project doesn't really have a good structure though and guards should probably be in a separate file with `index.ts` being a simple barrel file. But for now, I guess moving the symbol into a separate file altogether and then importing from that file into both `types.ts` and `index.ts` is a good enough solution.

Reply:
Sounds good to me.